src/ontology/cl-edit.owl

```diff
@@ -27748,18 +27730,15 @@ AnnotationAssertion(oboInOwl:inSubset obo:CL_4023042 <http://purl.obolibrary.org
 AnnotationAssertion(rdfs:label obo:CL_4023042 "L6 corticothalamic-projecting projecting glutamatergic cortical neuron")
 SubClassOf(obo:CL_4023042 obo:CL_4023013)
 
+# Class: obo:CL_4023046 (L6b subplate glutamatergic neuron of the primary motor cortex (Mus musculus))
 
-
-# Class: obo:CL_4023038 (L6b glutamatergic cortical neuron)
-
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2020.10.19.34312") Annotation(oboInOwl:hasDbXref "PMID:30382198") obo:IAO_0000115 obo:CL_4023038 "A glutamatergic neuron located in cortical layer 6b. They are transcriptomically related to corticothalamic-projecting neurons but have differential projections to the thalamus or anterior cingulate.")
-AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023038 <http://orcid.org/0000-0001-7258-9596>)
-AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023038 "ILX:0770163")
-AnnotationAssertion(oboInOwl:inSubset obo:CL_4023038 <http://purl.obolibrary.org/obo/cl#BDS_subset>)
-AnnotationAssertion(rdfs:label obo:CL_4023038 "L6b glutamatergic cortical neuron")
-SubClassOf(obo:CL_4023038 obo:CL_0000679)
-SubClassOf(obo:CL_4023038 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956))
-
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33184512") obo:IAO_0000115 obo:CL_4023046 "An excitatory glutamatergic neuron transcriptomically related to the CT subclass, preferentially located in the bottom of L6 of the primary motor cortex.")
```

Comment:
Just curious, are there any non-excitatory glutamatergic neurons or are all glutamatergic neurons excitatory?

Reply:
Honestly, I am not entirely sure. My undergrad knowledge wants to say yes, but at the same time I know that not all glutamate receptors for example are excitatory (like mGluRs) though that doesn't mean glut neurons arent excitatory. Glutamate can also be inhibitory in some cases (https://www.pnas.org/content/110/25/10294), though again, that does not necessarily mean that glutamate neurons themselves arent excitatory. Tried searching up on this and cant seem to get a direct answer too. I think they might be, but I am 100% sure that every single instance of a glutamatergic neuron is excitatory. (might also be overthinking this)

Reply:
Interesting -- I thought that every glutamatergic neuron was excitatory, but I'm no expert. 

I was curious because 'excitatory' occurs in this definition and I was wondering if it was redundant, i.e., if all glutamatergic neurons are excitatory, then there's no need to specify that this class of glutamatergic neurons are also excitatory.